telematic_system/historical_data_processing/rosbag2_processing_service/rosbag2_processing_service/service_manager.py

```diff
@@ -62,139 +55,69 @@ class ServiceManager:
-        if log_type == LogType.FILE.value:
-            self.log_handler = RotatingFileHandler(
-                self.log_path+log_name, maxBytes=self.log_rotation, backupCount=5)
-        else:
-             self.log_handler = logging.StreamHandler()
-        self.log_handler.setFormatter(formatter)
-
-        if(self.log_level == "debug"):
-            self.logger.setLevel(logging.DEBUG)
-            self.log_handler.setLevel(logging.DEBUG)
-        elif(self.log_level == "info"):
-            self.logger.setLevel(logging.INFO)
-            self.log_handler.setLevel(logging.INFO)
-        elif(self.log_level == "error"):
-            self.logger.setLevel(logging.ERROR)
-            self.log_handler.setLevel(logging.ERROR)
-
-        self.logger.addHandler(self.log_handler)
 
     async def nats_connect(self):
         """
             connect to nats server on EC2
         """
-
-        async def process_nats_request(msg):
-            self.logger.info("Entering process nats request")
+        # Nats request callback
+        async def get_file_path_from_nats(msg):
+            self.config.logger.info("Entering process nats request")
 
             data = msg.data.decode()
             msg_json_object = json.loads(data)
-            uploaded_path = msg_json_object["uploaded_path"]
+
             rosbag_name = msg_json_object["filename"]
 
-            rosbag_path = Path(uploaded_path) / rosbag_name
+            rosbag_path = Path(self.config.upload_destination_path) / rosbag_name
             # Add rosbag name to queue
             self.rosbag_queue.append(rosbag_path)
 
             # No response sent for the nats request
 
         async def disconnected_cb():
-            self.registered = False
-            self.logger.warn("Got disconnected...")
+            self.config.logger.warn("Got disconnected...")
 
         async def reconnected_cb():
-            self.logger.warn("Got reconnected...")
+            self.config.logger.warn("Got reconnected...")
 
         async def error_cb(err):
-            self.logger.error("{0}".format(err))
+            self.config.logger.error("{0}".format(err))
 
         if not self.is_nats_connected:
             try:
-                await self.nc.connect(self.nats_ip_port,
+                await self.nc.connect(self.config.nats_ip_port,
                                     reconnected_cb=reconnected_cb,
                                     disconnected_cb=disconnected_cb,
                                     error_cb=error_cb,
                                     max_reconnect_attempts=-1)
-                self.logger.info("Connected to NATS Server!")
+                self.config.logger.info("Connected to NATS Server!")
                 self.is_nats_connected = True
             finally:
-                self.logger.info("Client is trying to connect to NATS Server Done.")
+                self.config.logger.info("Client is trying to connect to NATS Server Done.")
```

Comment:
Why are we using `try:` without `except:`? This won't catch an raised errors. As it's implemented now, the `try: ... finally: ...` blocks are unnecessary.